### PR TITLE
Replacing classic entity mapping style for detection with new

### PR DESCRIPTION
**Pull request by d1mav0**
Required items, please complete
   
   Change(s):
   - Updated Detections\MultipleDataSources\AADHostLoginCorrelation.yaml

   Reason for Change(s):
   - New style of entity mapping

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Review by devikamehra — CHANGES_REQUESTED** [review]

File: Detections/MultipleDataSources/AADHostLoginCorrelation.yaml [hunk 1]
```diff
@@ -91,16 +91,16 @@ entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
-        columnName: AccountCustomEntity
```
[the remark this answers is not in the document]
Comment (d1mav0) [reply Detections/MultipleDataSources/AADHostLoginCorrelation.yaml:94]: Thanks for the comment! 

Where could I find the definition of what is currently supported for the analytics rule structure?

[the remark this answers is not in the document]
Comment (devikamehra) [reply Detections/MultipleDataSources/AADHostLoginCorrelation.yaml:94]: https://learn.microsoft.com/en-us/azure/sentinel/entities-reference

**Comment by v-atulyadav** [comment]
Hi @d1mav0, @devikamehra shared a link with you please modify rule accordingly. Thanks

**Comment by v-atulyadav** [comment]
Hi @d1mav0, looking forward to your response to the above comments. Thanks

**Comment by v-atulyadav** [comment]
Hi @d1mav0, we are still waiting for your response . Thanks

**Comment by v-atulyadav** [comment]
Hi @d1mav0, @devikamehra shared a link with you please modify rule accordingly. Thanks

**Comment by v-atulyadav** [comment]
Hi @d1mav0, I look forward to hearing back from you regarding the comments above. Thanks

**Comment by v-atulyadav** [comment]
Hi @d1mav0, we are still waiting for your response . Thanks

**Comment by v-atulyadav** [comment]
We wanted to check on the status of PR #8230. PR is pending for more than 30 days. Please let us know if you need any assistance to review this PR. Per our standard operating procedures if no response is received in the next 7 business days, we will close this PR. Thank you for your cooperation

**Comment by v-atulyadav** [comment]
Hi @d1mav0, Since we have not received a response in the last 7 days, we are closing your PR #8230 per our standard operating procedures. If you still need support for this issue, you can re-open the PR at any time. 
If you do re-open, we simply request that you ensure the PR has response to the last request. Thank you for your cooperation.

**Pull request closed without merging**